src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,57 @@
 import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+};
+
+export const MovieCard: React.FC<Props> = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
```

Comment:
Task says you have to pass `movie` prop instead of separate prop fields. So your tests did not pass.
Could you change your component interface?